datasets/us/sam_exclusions/us_sam_exclusions.yml

```diff
@@ -39,6 +39,18 @@ data:
 
 dates:
   formats: ["%m/%d/%Y"]
+names:
+  LegalEntity:
+    # Review names shorter than 11 chars, e.g. Sebastiano from G&G INTERNACIONAL S.A.S.
+    # We make Sebastiano an alias because it doesn't seem to be the primary official name of
```

Comment:
This comment made me think: Do we have any link from the review to the entity that it's for? Might be interesting to a) click through from reviews b) "fix up" a specific entity.

Out of scope here, just a thought.